api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -210,15 +211,6 @@ message HttpHeaders {
   // :ref:`value <envoy_v3_api_field_config.core.v3.HeaderValue.value>` field.
   config.core.v3.HeaderMap headers = 1;
 
-  // [#not-implemented-hide:]
-  // TODO(jbohanon) reserve field as part of rework detailed in https://github.com/envoyproxy/envoy/issues/32125
-  // The values of properties selected by the ``request_attributes``
-  // or ``response_attributes`` list in the configuration. Each entry
-  // in the list is populated
-  // from the standard :ref:`attributes <arch_overview_attributes>`
-  // supported across Envoy.
-  map<string, google.protobuf.Struct> attributes = 2;
```

Comment:
Please don't remove this directly. Just keep it and mark it as deprecated.

Implementation LGTM overall. but I just do a quick check on the phone, so will check it on PC tomorrow.

Reply:
Although specific to this field, I agree with you, there should be no risk and removing make the API more clean.

But our API policy tell:

> Deprecations will still occur as an end-user indication that there is a preferred way to configure a particular feature, but **no field will ever be removed** nor will Envoy ever remove the implementation for any deprecated field.

The policy is desigend for common cases, it may not always be the best choice in every special case. But rule is the rule. 
So, ...

Reply:
Sounds good. I went ahead and added it back with the `[#not-implemented-hide:]` and marked it deprecated.